src/main/java/com/chess/backend/gamemodel/Move.java

```diff
@@ -13,8 +15,7 @@
     protected Moves.castling castlingMove = Moves.castling.none;
     protected boolean wasPawnTwoFieldsMove = false;
 
-    public Move(Square from, Square to, Piece movedPiece, Piece takenPiece, Moves.castling castlingMove, boolean wasEnPassant, Piece promotedPiece)
-    {
```

Comment:
I think all of you have to choose a code-formater in the backend, so you don't have to take of that manually

Reply:
Oh shit good point, will make a code-format commit shortly. Thanks!